internal/accounts/sql.go

```diff
@@ -27,15 +29,18 @@ const (
 	sqlGetAccount = `SELECT balance, nonce FROM ` + schemaName + `.accounts WHERE identifier = $1`
 )
 
-func initTables(ctx context.Context, tx sql.DB) error {
-	if _, err := tx.Execute(ctx, sqlCreateSchema); err != nil {
-		return err
+func initTables(ctx context.Context, db sql.DB) error {
+	tx, err := db.BeginTx(ctx)
```

Comment:
`InitializeAccountStore` also wraps with a tx, then `Upgrade` wraps with a tx, and a third here in `initTables`.  I can't tell why more than one transaction is needed on that path.